main/getting-started/start-a-project.md

```diff
@@ -35,70 +32,36 @@ This presentation includes starting a project, but note an outdated detail:
 Use the [Agoric CLI](/guides/agoric-cli/commands.md) to fetch from a Dapp template
 and put it in a `demo` directory _not located in your `agoric-sdk` clone_:
 
-
 ```sh
 # Terminal 1
-# Don't use your agoric-sdk clone as the parent of the demo directory.
-# It doesn't have to be your home directory; any other directory will do.
+# Don't use your agoric-sdk as the parent of the demo directory.
 cd $HOME
-agoric init demo
+agoric init demo # use `agoric init $DIRNAME` with any name you like
+cd demo
+agoric install
+agoric start --verbose # `agoric start --reset` to start over
 ```
 
-The name `demo` is an arbitrary suggestion; in general,
-use `agoric init $DIRNAME` with any name you like.
+The name `demo` is an arbitrary suggestion. The default template is the [Fungible Faucet Dapp](https://github.com/Agoric/dapp-fungible-faucet).
```

Comment:
Seems like a non-sequitor. The fact that other names can be used in place of `demo` is unrelated to the choice of dapp.

Reply:
broke that up a little in next push